fusor_ovirt.spec

```diff
@@ -0,0 +1,37 @@
+Name:           fusor_ovirt
+Version:        0.1
+Release:        1%{?dist}
+Summary:        Scripts to interact with oVirt API
+
+Group:          Applications/System
+License:        GPLV3+ and ASL 2.0
+URL:            https://github.com/fusor/fusor_ovirt
+Source0:        %{name}-%{version}.tar.gz
+
+Requires:       ovirt-engine-sdk-python
+
+%description
+Python scripts to interact with the oVirt API
+
+%prep
+%setup -q
+
+
+%build
+
+%install
+install -d -m0755 %{buildroot}%{_bindir}
+cp bin/*.py %{buildroot}%{_bindir}
```

Comment:
Ok spoke with @bbuckingham, let's move them to ```/usr/share/fusor/bin```. That way they won't be accidentally run by users from their path, and we have a stable path to put in the project code to find them.